seed_data/challenges/computer-science.json

```diff
@@ -90,8 +103,12 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende sobre Redes de Computadoras",
+      "descriptionEs": [
+        "Ahora que ya hemos aprendido sobre, es hora de aprender sobre el software que corre encima de ello.",
```

Comment:
This paragraph has a huge mistake: it doesn't contain the word "hardware" so it doesn't make sense.

It should say:
"Ahora que ya aprendimos sobre hardware, es hora de aprender sobre el software que se ejecuta arriba de él."